EPRN.Portal/Areas/Exporter/Controllers/PRNSController.cs

```diff
@@ -1,29 +1,35 @@
 ﻿using EPRN.Common.Constants;
 using EPRN.Common.Enums;
 using EPRN.Portal.Controllers;
+using EPRN.Portal.Helpers.Filters;
 using EPRN.Portal.Services.Interfaces;
 using EPRN.Portal.ViewModels.PRNS;
+using EPRN.Portal.ViewModels.Waste;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
 using static EPRN.Common.Constants.Strings;
 
 namespace EPRN.Portal.Areas.Exporter.Controllers
 {
     [Area(Routes.Areas.Exporter)]
+    [ServiceFilter(typeof(WasteTypeActionFilter))]
```

Comment:
WasteType action filter doesn't seem appropriate for PRN controller. Are we sure this is correct?

Reply:
If you recall originally we tacked on the user reference to this action filter. However, we should really create another mechanism to store user data. Open to suggestions.

Reply:
Unfortunately, we need it to save / get records for certain scenarios. If we let this go through for now, I'll create a Middleware piece of code which can add user info to the HttpContext ... but will do that in another story.